package.json

```diff
@@ -2,25 +2,24 @@
 	"name": "data-update-github",
 	"version": "0.1.0",
 	"description": "A remote data change triggers the issue of a pull request to a specified Github repo.",
+	"main": "index.js",
 	"scripts": {
-		"lint": "grunt lint",
+		"lint": "fluid-lint-all",
 		"test": "node tests/all-tests.js"
 	},
 	"author": "Inclusive Design Research Centre",
 	"license": "BSD-3-Clause",
 	"dependencies": {
-		"axios": "0.20.0",
-		"eslint-config-fluid": "1.4.0",
-		"fluid-grunt-lint-all": "2.0.0",
-		"jsdom": "16.4.0",
-		"json5": "2.1.3",
-		"rimraf": "3.0.2",
-		"simple-git": "2.21.0"
+		"@octokit/core": "3.5.1",
+		"axios": "0.21.1",
+		"json5": "2.2.0"
 	},
 	"devDependencies": {
-		"grunt": "1.3.0",
+		"dotenv": "10.0.0",
+		"eslint-config-fluid": "2.0.0",
+		"fluid-lint-all": "1.1.4",
 		"infusion": "3.0.0-dev.20201113T153152Z.32176dcbe.FLUID-6145",
```

Comment:
Could you also update to a modern Infusion release such as 4.0.0-dev.20210622T193827Z.ac6adb829.FLUID-6629 - thanks!